app/views/events/_form.html.erb

```diff
@@ -69,4 +69,8 @@
   <div class="form-group">
     <%= form.submit :Submit, class: 'btn btn-primary' %>
   </div>
```

Comment:
I think the back and submit button would look better on the same line!  Pretty simple fix.  

<div class="form-group" style= "display:inline-block;">
    <%= form.submit :Submit, class: 'btn btn-primary' %>
    <%= link_to 'Back', :back, class: "btn btn-secondary" %>
  </div>